plugins/chooseYourAdventurePlayer/chooseYourAdventurePlayer.yml

```diff
@@ -0,0 +1,45 @@
+name: Choose Your Own Adventure Player
+description: Plugin to adapt Choose Your Own Adventure Games to the VideoJS player.
+version: 1.0.1
+# requires: CommunityScriptsUILibrary
+settings:
+  accessToken:
+    displayName: Access Token
+    description: Your personal Github access token needed to read private scripts
+    type: STRING
+  gameTag:
+    displayName: Game tag
+    description: Name of the tag indicating a scene is the starting point of game. Player will initialize once scene with this tag is loaded. Only use this tag on the starting scene.
+    type: STRING
+  enableGameSave:
+    displayName: Enable game save
+    description: When enabled you will be giving the opportunity to resume your games from where you last left off which all your choices in tack.
+    type: BOOLEAN
+  resourcePath:
+    displayName: Resource path
+    description: Location where game resources live (images, jsons). It is recommended to create custom served folder for this.
+    type: STRING
+  rootScenesPath:
+    displayName: Root scenes directory path
+    description: Data path where game scenes live.
+    type: STRING
+  useCustomPosters:
+    displayName: Use custom game posters
+    description: When enabled the t=start game screen will use provided poster instead of default scene cover. Poster should be named with the following comvention {resourcePath}/{gameId}/poster_{number}.jpg
+    type: BOOLEAN
+ui:
+  requires:
+    - CommunityScriptsUILibrary
+  javascript:
+    - https://cdn.jsdelivr.net/gh/HandyRandyx/stash-plugins@main/utils/fetchInterceptor.js
+    - https://cdn.jsdelivr.net/gh/HandyRandyx/stash-plugins@main/utils/stashHandler.js
+    - https://cdn.jsdelivr.net/gh/HandyRandyx/stash-plugins@main/utils/registerPathChangeListener.js
+    - https://cdn.jsdelivr.net/gh/cj12312021/stash-plugins@main/utils/waitForClass.js
+    - https://vjs.zencdn.net/8.16.1/video.min.js
+    - https://raw.githubusercontent.com/chrisboustead/videojs-vtt-thumbnails/master/src/plugin.js
+    - cyap-min.js
+  css:
+    - chooseYourAdventurePlayer.css
```

Comment:
Missing source file.

Reply:
See my post here: https://discordapp.com/channels/559159668438728723/742220889017417738/1333190984070205460. I intend to keep the JS file behind a private repo meant for members only. I want to keep the plugin on the script page with instructions on attaining pull access to make maker backer benefits visible.

Reply:
Please hold off on merging this PR as I sort some details out with [stg-annon](https://github.com/stg-annon). The instruction here will need another pass.

Reply:
That was my understanding with .js file, but does the same is true for the missing .css file? GitHub didn't allow me to highlight a single line.

Reply:
Yeah, I considered including the CSS file, but I ultimately decided against it because providing it would not be of any real benefit.

I just pushed more changes to the branch. I think it's in a good state for an initial release.